src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -444,7 +453,8 @@ End Class
             End Using
         End Function
 
-        <MemberData(NameOf(AllCompletionImplementations))> <WpfTheory, Trait(Traits.Feature, Traits.Features.Completion)>
+        <MemberData(NameOf(AllCompletionImplementations))>
+        <WpfTheory, Trait(Traits.Feature, Traits.Features.Completion)>
         Public Async Function TestNavigationBeforeCompletedComputation(completionImplementation As CompletionImplementation) As Task
```

Comment:
❗️ Add a copy of this test using the same `SendMoveToPreviousCharacter` as the current test, but trigger the completion session using `SendTypeChars(".")` instead of the current `SendTypeChars(".Ma")`. Since the move to the previous character moves to a different context, this test case should also not have a completion session.

Reply:
[https://github.com/dotnet/roslyn/issues/30970](https://github.com/dotnet/roslyn/issues/30970)

---
In reply to: [229269015](https://github.com/dotnet/roslyn/pull/29016#discussion_r229269015) [](ancestors = 229269015)